internal/server/flag.go

```diff
@@ -16,10 +16,6 @@ import (
 func (s *Server) GetFlag(ctx context.Context, r *flipt.GetFlagRequest) (*flipt.Flag, error) {
 	s.logger.Debug("get flag", zap.Stringer("request", r))
 
-	if r.NamespaceKey == "" {
-		r.NamespaceKey = storage.DefaultNamespace
-	}
```

Comment:
Just curious, I see a bunch of places where you have removed this. Is this a case of the storage layer is handling this? 

I saw you mentioned that this was missing from GetEvaluationRollouts and see you added it there, but removed it here.

Reply:
i moved it from the server layer, to me it makes sense to have this check in the storage layer. For the changes in the storage layer I simply bumped it up to be the first line in the methods so its easy to see if its there or not.

I think it might make sense for us to create a middleware for the server layer to ensure the namespace is set or set the default as well